src/vscode-dts/vscode.proposed.inlineEdit.d.ts

```diff
@@ -0,0 +1,89 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+declare module 'vscode' {
+	export class InlineEdit {
+		/**
+		 * The position of the edit.
+		 */
+		position: Position;
+
+		/**
+		 * The new text for this edit.
+		 */
+		text: string;
+
+		/**
+		 * An optional range that will be replaced by the text of the inline edit.
+		 */
+		replaceRange?: Range;
```

Comment:
What if `replaceRange` does not agree with `position`? Maybe there should just be a required `replaceRange` (which can be empty)?

Reply:
That's a good idea 👍 

I've changed the API just to be required `range` and removed `position`